app/src/main/java/com/celzero/bravedns/util/Utilities.kt

```diff
@@ -250,6 +260,36 @@ class Utilities {
             } else String.format(Locale.ROOT, "%d", type)
         }
 
+        fun getThemeAccent(context: Context): Int {
+            return if (isDarkSystemTheme(context)) {
+                R.color.accentGoodBlack
+            } else {
+                R.color.negative_white
+            }
+        }
+
+        private fun isDarkSystemTheme(context: Context): Boolean {
+            return context.resources.configuration.uiMode and Configuration.UI_MODE_NIGHT_MASK == Configuration.UI_MODE_NIGHT_YES
+        }
+
+        fun openVPNProfile(context: Context) {
+            try {
+                val intent = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+                    Intent(ACTION_VPN_SETTINGS)
+                } else {
+                    Intent("android.net.vpn.SETTINGS")
+                }
+                intent.flags = Intent.FLAG_ACTIVITY_NEW_TASK
+                context.startActivity(intent)
+            } catch (e: ActivityNotFoundException) {
+                showToastInMidLayout(context, context.getString(R.string.vpn_profile_error), Toast.LENGTH_SHORT)
```

Comment:
rename?

`showToastUiCentered` or something better. `in` isn't really required.

Reply:
Renamed to `showToastUiCentered`